index.html

```diff
@@ -9,96 +9,125 @@
+                  />
+                </label>
+                <button class="search-btn" type="submit">Найти</button>
+              </form>
+              <ul class="user-nav__list">
+                <li class="user-nav__item">
+                  <a class="user-nav-link" href="#">
+                    <span class="icon-login">Войти</span>
                   </a>
                 </li>
-                <li class="other-item">
-                  <a class="other-link" href="catalog.html"
-                    >Моноподы для cелфи
+                <li class="user-nav__item">
+                  <a class="user-nav-link" href="#">
+                    <span class="icon-compare">Сравнить</span>
                   </a>
                 </li>
-                <li class="other-item">
-                  <a class="other-link" href="catalog.html">Экшн-камеры</a>
-                </li>
-              </ul>
-              <ul class="clock-list">
-                <li class="clock-item">
-                  <a class="clock-link" href="catalog.html">Фитнес-браслеты</a>
-                </li>
-                <li class="clock-item">
-                  <a class="clock-link" href="catalog.html">Умные часы</a>
-                </li>
-              </ul>
-              <ul class="quadcopters-list">
-                <li class="quadcopters-item">
-                  <a class="quadcopters-link" href="catalog.html"
-                    >Фитнес-браслеты
+                <li class="user-nav__item">
+                  <a class="user-nav-link" href="#">
+                    <span class="icon-basket">Корзина</span>
                   </a>
                 </li>
               </ul>
             </div>
-          </div>
-          <nav class="nav-page">
-            <ul class="nav-page__list">
-              <li class="nav-page__item">
-                <a class="nav-page__link" href="#">доставка</a>
-              </li>
-              <li class="nav-page__item">
-                <a class="nav-page__link" href="#">гарантия</a>
-              </li>
-              <li class="nav-page__item">
-                <a class="nav-page__link" href="#">контакты</a>
-              </li>
-            </ul>
-          </nav>
+            <section class="page-nav-container">
+              <h2 class="visually-hidden">Каталог товаров</h2>
+              <div class="nav-catalog">
+                <a class="catalog-link link" href="catalog.html">
+                  каталог товаров
+                </a>
+                <button class="btn-catalog">
+                  <span class="visually-hidden">Перейти к каталогу</span>
+                </button>
+                <div class="catalog-group">
+                  <div class="catalog-group-inner">
+                    <ul class="other-list">
+                      <li class="other-item">
+                        <a class="other-link" href="#"
+                          >Виртуальная реальность
+                        </a>
```

Comment:
Опять проблема со ссылками
```suggestion
                        <a class="other-link" href="#">Виртуальная реальность</a>
```